docs/README.md

```diff
@@ -1,18 +1,19 @@
 # Introduction to Starport
 
-Starport is a developer-friendly interface to the Cosmos SDK.
+Starport offers everything you need to build, test, and launch your blockchain with a decentralized worldwide community. Starport is built on top of [Cosmos SDK](https://docs.cosmos.network), the world’s most popular blockchain framework. Starport accelerates chain development by scaffolding everything you need so you can focus on business logic.
```

Comment:
**:praise:** Excellent and catchy. Let's get more people building more blockchains.

Reply:
I actually took it from the website 🙈